Sources/TuistDependencies/SwiftPackageManager/Models/WorkspaceState.swift

```diff
@@ -12,6 +12,7 @@ struct SwiftPackageManagerWorkspaceState: Decodable, Equatable {
 
     struct Dependency: Decodable, Equatable {
         let packageRef: PackageRef
+        let subpath: String
```

Comment:
Nitpick: I'd add a comment here explaining why `subpath` is necessary. That's a context that you captured on the PR, but I think it'll be useful if we capture it here too.